src/MeCab/MeCab.i

```diff
@@ -26,8 +26,8 @@
 %rename(DictionaryInfo) mecab_dictionary_info_t;
 %ignore    mecab_model_t;
 %ignore    mecab_lattice_t;
-%nodefault mecab_path_t;
-%nodefault mecab_node_t;
+%nodefaultctor mecab_path_t;
+%nodefaultctor mecab_node_t;
```

Comment:
I do not understand SWIG well enough to know whether this is the correct fix for the reported memory leak.  I'll accept this part of the change if @polm believes it to be correct.

Reply:
Original line directed to omit generation of both constructor and destructor for the given objects. While constructors should be avoided (since objects are returned and constructed by MeCab), the destruction is still needed. I debugged generated code in both cases. Before the change, the python object (the one which wraps the one returned by MeCab) is not released causing memory leak. After the change, SWIG generated destructor, releases python object leaving MeCab object untouched (it checks `own` flag) - the behavior we want.

Reply:
I understand that this is the effect that this change has.  What I don't know is whether that is the correct effect.  Someone who understands SWIG better than me needs to review the change before I can accept it.

Reply:
I am not sure if we can find someone here who knows SWIG that much. But if you understands Python interop APIs, that might be a good alternative for understanding this change.

Reply:
@polm is happy, so that's good enough for me.